Sprint-3/revise/implement/get-ordinal-number.js

```diff
@@ -7,25 +7,25 @@ function getOrdinalNumber(number) {
   const numberAsString = number.toString();
 
   // Helper variable
-  const exceptions = ["11", "12", "13"];
+  const exceptions = [11, 12, 13];
+  const lastDgit = number % 10;
+  const lastTwoDigits = number % 100;
 
-  if (
-    numberAsString.length > 1 &&
-    exceptions.includes(numberAsString.slice(numberAsString.length - 2))
-  ) {
+  if (number > 9 && exceptions.includes(lastTwoDigits)) {
```

Comment:
What numbers don't have two digits remainder?

Reply:
Ok, ok, ok, I see what you saying now. It took me a minute to click but I see what you mean now.

I've updated the if statement `if (exceptions.includes(number)) {` and I've removed the `lastTwoDigits` variable.